src/config/settings.py

```
@@ -0,0 +1,72 @@
+"""
+Configuration settings for the AI backtesting system.
+"""
+import os
+from typing import List
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)